app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -92,11 +90,17 @@ class MainActivity : AppCompatActivity() {
 
     private val visionEventsListener = object : VisionEventsListener {
 
+        override fun onAuthorizationStatusUpdated(authorizationStatus: AuthorizationStatus) {
+            println("Auth $authorizationStatus")
```

Comment:
you'd like to have system out print here. Are you sure?

Reply:
Nope, sure, will remove, debug trash(